### PR TITLE
replacing rounded faraday constant with exact value in bpx.py

### PR DESCRIPTION
**Pull request by dion-w**
# Description

The faraday constant has an exact value, calculated by the exact values of the avogadro constant and the elementray charge [[see here](https://www.nist.gov/si-redefinition/meet-constants)]. Using a rounded value may introduce systemic errors further down the line that can be easily avoided with this new value.

Fixes # (issue)

## Type of change

Please add a line in the relevant section of [CHANGELOG.md](https://github.com/pybamm-team/PyBaMM/blob/develop/CHANGELOG.md) to document the change (include PR #) - note reverse order of PR #s. If necessary, also add to the list of breaking changes.

- [ ] New feature (non-breaking change which adds functionality)
- [ ] Optimization (back-end change that speeds up the code)
- [x] Bug fix (non-breaking change which fixes an issue)

# Key checklist:

- [x] No style issues: `$ pre-commit run` (or `$ nox -s pre-commit`) (see [CONTRIBUTING.md](https://github.com/pybamm-team/PyBaMM/blob/develop/CONTRIBUTING.md#installing-and-using-pre-commit) for how to set this up to run automatically when committing locally, in just two lines of code)
- [x] All tests pass: `$ python run-tests.py --all` (or `$ nox -s tests`)
- [x] The documentation builds: `$ python run-tests.py --doctest` (or `$ nox -s doctests`)

You can run integration tests, unit tests, and doctests together at once, using `$ python run-tests.py --quick` (or `$ nox -s quick`).

## Further checks:

- [x] Code is commented, particularly in hard-to-understand areas
- [not needed] Tests added that prove fix is effective or that feature works

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -4,6 +4,7 @@
 
 ## Features
 
+- replaced rounded faraday constant with it's exact value in bpx.py for better comparison between different tools
```
Comment (agriyakhetarpal) [inline CHANGELOG.md:7]:
```suggestion
- Replaced rounded faraday constant with its exact value in `bpx.py` for better comparison between different tools
```

Comment (agriyakhetarpal) [reply CHANGELOG.md:7]: Additionally, could you please move it to the unreleased section, @dion-w?

Comment (dion-w) [reply CHANGELOG.md:7]: yeah sure

**Review by arjxn-py — APPROVED** [review]
Thanks @dion-w, looks good

**Comment by rtimms** [comment]
FYI you can do `pybamm.constants.F` or `pybamm.constants.F.value` if you need a number instead of a symbol.

**Comment by kratman** [comment]
> FYI you can do `pybamm.constants.F` or `pybamm.constants.F.value` if you need a number instead of a symbol.

I agree, using a constant from a library is better than updating this file.

One other thing to check would be if the BPX standard specifies a value for Faraday constant. Sometimes standards specify the precision to ensure that things are constant between implementations

File: pybamm/parameters/bpx.py [hunk 2]
```diff
@@ -240,7 +240,7 @@ def _entropic_change(sto, c_s_max, dUdT, constant=False):
     # (cs-cs_max)) in BPX exchange current is defined j0 = F * k_norm * sqrt((ce/ce0) *
     # (cs/cs_max) * (1-cs/cs_max))
     c_e = pybamm_dict["Initial concentration in electrolyte [mol.m-3]"]
-    F = 96485
+    F = 96485.3321233100184
```
Comment (kratman) [inline pybamm/parameters/bpx.py:243]: If the BPX spec does not specify a value, then this is better
```suggestion
    F = pybamm.constants.F.value
```

Note: I did not test this myself

**Review by kratman — CHANGES_REQUESTED** [review]

**Comment by agriyakhetarpal** [comment]
Re-requested review/approval from @kratman since he previously requested changes

**Review by kratman — APPROVED** [review]

**Pull request merged**